express-notes-api/index.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const fs = require('fs');
+const jsonData = require('./data.json');
+const app = express();
+
+app.get('/api/notes', (req, res) => {
+  const notesArr = [];
+  for (const id in jsonData.notes) {
+    notesArr.push(jsonData.notes[id]);
+  }
+  res.status(200).json(notesArr);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  let notesObj = {};
+  if (req.params.id < 0) {
+    res.status(400).json({ error: 'id must be a positive integer' });
+  } else if (jsonData.notes[req.params.id]) {
+    notesObj = jsonData.notes[req.params.id];
+    res.status(200).json(notesObj);
+  } else if (jsonData.notes[req.params.id] === undefined) {
```

Comment:
Here you can just use an `else` you already checked that `jsonData.notes[req.params.id]` has a value so you don't need to check the exact opposite of what you already checked.

Reply:
Thank you, Scott!  An `else` would cover so much more than just one specific condition